infra/capi/cluster-capi-infrastructure-metal3/Kptfile

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: cluster-capi-infrastructure-metal3
```

Comment:
Does it matter that the file is now "cluster-api-infrastructure-metal3.yaml" instead of "cluster-capi-infrastructure-metal3.yaml"